src/api/routes/authorsRoutes.js

```diff
@@ -0,0 +1,9 @@
+import { Router } from 'express';
+import getAuthorsController from '../controllers/getAuthorsController';
+import tokenVerifier from '../../middlewares/tokenVerifier';
```

Comment:
Your team-mates already have a similar middleware for token verification @Ramadhan0... sync with them